codemeta.json

```diff
@@ -0,0 +1,295 @@
+        {
+            "@type": "Person",
+            "givenName": "Konrad",
+            "familyName": "Patkowski",
+            "affiliation": "Department of Chemistry and Biochemistry, Auburn University",
+            "@id": "https://orcid.org/0000-0002-4468-207X",
+            "@id": "https://github.com/konpat"
+        },
+        {
+            "@type": "Person",
+            "givenName": "Rollin",
+            "additionalName": "A.",
+            "familyName": "King",
+            "affiliation": "Department of Chemistry, Bethel University",
+            "@id": "https://orcid.org/0000-0002-1173-4187",
+            "@id": "https://github.com/psi-rking"
+        },
+        {
+            "@type": "Person",
+            "givenName": "Edward",
+            "additionalName": "F.",
+            "familyName": "Valeev",
+            "affiliation": "Department of Chemistry, Virginia Tech",
+            "@id": "https://orcid.org/0000-0001-9923-6256",
+            "@id": "https://github.com/evaleev"
+        },
+        {
+            "@type": "Person",
+            "givenName": "Francesco",
+            "additionalName": "A.",
+            "familyName": "Evangelista",
+            "affiliation": "Department of Chemistry, Emory University",
+            "@id": "https://orcid.org/0000-0002-7917-6652",
+            "@id": "https://github.com/fevangelista"
+        },
+        {
+            "@type": "Person",
+            "givenName": "Justin",
+            "additionalName": "M.",
+            "familyName": "Turney",
+            "affiliation": "Center for Computational Quantum Chemistry, University of Georgia",
+            "@id": "https://orcid.org/0000-0003-3659-0711",
+            "@id": "https://github.com/jturney"
+        },
+        {
+            "@type": "Person",
+            "givenName": "T.",
+            "additionalName": "Daniel",
+            "familyName": "Crawford",
+            "affiliation": "Department of Chemistry, Virginia Tech",
+            "@id": "https://orcid.org/0000-0002-7961-7016",
+            "@id": "https://github.com/lothian"
+        },
+        {
+            "@type": "Person",
+            "givenName": "C.",
+            "additionalName": "David",
+            "familyName": "Sherrill",
+            "affiliation": "Center for Computational Molecular Science and Technology, Georgia Institute of Technology",
+            "@id": "https://orcid.org/0000-0002-5570-7666",
+            "@id": "https://github.com/CDSherrill"
+        },
+        {
+            "@type": "Person",
+            "givenName": "Peter",
+            "familyName": "Kraus",
+            "affiliation": "Liebniz University, Hannover, Germany",
+            "@id": "https://orcid.org/0000-0002-4359-5003",
+            "@id": "https://github.com/PeterKraus"
+        },
+        {
+            "@type": "Person",
+            "givenName": "Holger",
+            "familyName": "Kruse",
+            "affiliation": "Institute of Biophysics, Czech Republic",
```

Comment:
please add the full name, there might be others: _Institute of Biophysics of the CAS, v.v.i., Czech Republic_

Reply:
out of curiosity, what's "v.v.i."?

Reply:
I believe it stands for 'public research institution' (in czech). Don't ask why the English name has it...